### ISSUE TITLE
bug: Check for updates from the "About" window gives a connection error

### ISSUE THREAD

**Issue by Ho-Ro** [NanoVNA-Saver/nanovna-saver#638]
# Bug Report

**NanoVNA-Saver version: 0.5.5

**Current behavior:**

If you're using the latest stable release 0.5.5 then the check for updates from the `About` window gives a connection error due to changed URLs (the version check looks directly into the github source code of module `About.py`).

**Expected behavior:**

Version check

**Other information:**

https://groups.io/g/nanovna-users/message/32908

**Comment by Ho-Ro** [comment]
https://github.com/NanoVNA-Saver/nanovna-saver/blob/ce8a59d478fe17365f68ede28c3c3557c04ed53b/src/NanoVNASaver/About.py#L26-L29

The URL is wrong also in the latest source code, `master` has changed to `main`, it can also be shorter:
```
VERSION_URL = "https://github.com/NanoVNA-Saver/nanovna-saver/main/src/NanoVNASaver/About.py"
```

Then it fits into the line of the changed error message:

https://github.com/NanoVNA-Saver/nanovna-saver/blob/ce8a59d478fe17365f68ede28c3c3557c04ed53b/src/NanoVNASaver/Windows/About.py#L132-L137

```
            self.updateLabel.setText(f"{e}:\n{VERSION_URL}")
```

**Comment by Ho-Ro** [comment]
fixed with #647
